### ISSUE TITLE
Artifact-rules: Incorrect warning emitted for ACS AEM Commons

### ISSUE THREAD

**Issue by kwin** [adobe/aemanalyser-maven-plugin#285]
With ACS AEM Commons 6.4.0 being embedded in a container content package and being checked with `aemanalyser:1.6.4:project-analyse` the following incorrect warning is emitted:

`[WARNING] [artifact-rules] com.adobe.acs:acs-aem-commons-bundle:6.4.0: Use at least version 5.0.10 for ACS AEM Commons (my-group/common-3rd-party-container:1.0.0-SNAPSHOT|adobe/consulting:acs-aem-commons-all:6.4.0)`

Obviously the version comparison is done incorrectly against the used package version of ACS AEM Commons.

**Comment by kwin** [comment]
This is coming from an incorrect artifact-rule within aem-sdk-api (e.g. in https://repo1.maven.org/maven2/com/adobe/aem/aem-sdk-api/2024.7.17258.20240726T172406Z-240700/aem-sdk-api-2024.7.17258.20240726T172406Z-240700-aem-author-sdk.slingosgifeature)

```
 "artifact-rules:JSON|false":{
    "mode":"LENIENT",
    "bundle-version-rules":[
      {
        "artifact-id":"com.adobe.acs:acs-aem-commons-bundle:0",
        "message":"Use at least version 5.0.10 for ACS AEM Commons",
        "allowed-version-ranges":[
          "[5.0.10,6.0.0)"
        ]
      },
```

There shouldn't be an upper bound in that rule!

**Comment by kwin** [comment]
This is fixed now in https://repo1.maven.org/maven2/com/adobe/aem/aem-sdk-api/2024.8.17569.20240822T203847Z-240700/aem-sdk-api-2024.8.17569.20240822T203847Z-240700-aem-author-sdk.slingosgifeature. However it again has an upper bound (for no obvious reason) which will be violated soon:

```
"artifact-rules:JSON|false":{
    "mode":"LENIENT",
    "bundle-version-rules":[
      {
        "artifact-id":"com.adobe.acs:acs-aem-commons-bundle:0",
        "message":"Use at least version 6.0.0 for ACS AEM Commons",
        "allowed-version-ranges":[
          "[6.0.0,7.0.0)"
        ]
      },
```

@cziegeler Can you please remove the upper bound? 7.0.0 appears soonish...

**Comment by cziegeler** [comment]
@kwin Lets wait until 7 is out because I also want to update the message to drive users to update to 7
